src/components/form/Lookup/LookupField.tsx

```diff
@@ -105,9 +106,10 @@ export function LookupField<
   getCompareBy,
   getOptionLabel: getOptionLabelProp,
   variant,
+  canCreate = false,
```

Comment:
Let's move the power check inside of this component. Pulling this boolean shouldn't be a responsibility of the callers of this component. It's already opinionated and aware of our API anyways.

The `createFor` could default it as well.
```tsx
<LookupField<...>
  ...
  createPower={`Create${resource}`}
  {...config}
  {...props}
/>
```

Reply:
It has to be set in `createFor` right?  How else will the component know what power to check?

Reply:
Right. The hook will still need to be called in the LookupField component.

Reply:
Updated.  Is there a way to make this resource type more strict?  Even if we manually create a list of Resource enums...  A misspelling will cause the powers check to not work

Reply:
Yeah I wish there was. I think it might be possible with TS 4.1. For now it's ok.